### ISSUE TITLE
Skip to links missing on wiki pages

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#82]
While the Skip to ... links are still present and functional on the landing page, they are missing from the wiki pages.

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563466
